### ISSUE TITLE
After click a button in one2many lines, the page scrolls to top.

### ISSUE THREAD

**Issue by tanghulu0608** [odoo/odoo#50356]
**Impacted versions:**
13.0

**Steps to reproduce:**
1. A stock picking with a lot of move lines.
![image](https://user-images.githubusercontent.com/5561864/80557012-11217c00-8a08-11ea-91d5-ce6d263bada3.png)
2. User want to set the lot number of the move at the bottom, the user scroll the page to the bottom, click the set lot button.
3. User set the lot number, and click the confirm button
![image](https://user-images.githubusercontent.com/5561864/80557192-b2a8cd80-8a08-11ea-9cd4-e5b2f76a8967.png)
4. when the user come back to stock picking form view, the view is scroll to top.
5. User have to scroll to bottom again the set lot number of other move line.
6. User is scroll to bottom again and again.

**Expected behavior:**
After click a button in one2many lines, the page **do not** scrolls to top, just stay where it is.

**Comment by pedrobaeza** [comment]
I'm afraid that's currently something not feasible due to the technical way this is done. You can check detailed operations in the picking type for avoiding the extra popup and then the refresh.

Any way, I mark this as wishlist for the web client team.

**Comment by tanghulu0608** [comment]
Thanks. I think this very useful.Stock picking is just an example.

**Comment by Yenthe666** [comment]
Yeah this was discussed before (I believe with @ged-odoo) that right now the behaviour is that it goes to the top but that it would be great to keep the height in future versions.
@ged-odoo do you remember where we've discussed this?

**Comment by tanghulu0608** [comment]
@ged-odoo  hello

**Comment by tanghulu0608** [comment]
@ged-odoo  ping.

**Comment by tanghulu0608** [comment]
@tommysnello  as far as I know, no work round yet.
